Classes.py

```diff
@@ -0,0 +1,209 @@
+
+        self.mPropImpulse   = mPropImpulse
+        self.mPropImpulseOx = mPropImpulseOx
+        self.mPropImpulseFuel = mPropImpulseFuel
+ 
+# This class creates a summary of the entire mission - summing things up across
+# phases         
+class MissionSummary:
+    def __init__(self, tupPhases):
+        """
+        Inputs:
+            tupPhases: list of phase classes
+        
+        """
+        # Initialize variables 
+        mPropImpulse     = 0
+        mPropImpulseOx   = 0
+        mPropImpulseFuel = 0
+
+        # sum up the usages by phase
+        for curPhase in tupPhases:
+            mPropImpulse     += curPhase.mPropImpulse 
+            mPropImpulseOx   += curPhase.mPropImpulseOx
+            mPropImpulseFuel += curPhase.mPropImpulseFuel
+
+        # Stuff everything into self    
+        self.mPropImpulse      = mPropImpulse
+        self.mPropImpulseOx    = mPropImpulseOx
+        self.mPropImpulseFuel  = mPropImpulseFuel
+
+
+# This builds up an engine, with an assumed thrust, specific impulse, and 
+# mixture ratio
+class Engine:
+    def __init__(self,isp, thrust, mr):
+        self.isp = isp
+        self.thrust = thrust
+        self.mr = mr
+
+
+class TankSet:
+    def __init__(self, strPropType,strMatType, nTanks, lMaxRadTank, presTank, mPropTotal):
+        # General Parameters for Tanks
+        pctUllage =  .10        # Extra ullage room as a percentage of tank volume
+        aMax      =   50        # Maximum acceleration (m/s2)
+        pctFudge  =    1.2     # Fudge factor for welds, etc
+        fosMat    =    1.5      # factor of safety for material (nd)
+        
+        # Tank material switch case
+        if strMatType=="Al2219":
+            rhoMat =   2840   # Density of material (kg/m3)
+            sigMat = 2.9e8    # Yield Stress of material (Pa)
+            thkMin =  .004  # Minimum thickness of material (m)
+        elif strMatType=="Stainless":
+            rhoMat = 8000
+            sigMat = 2.15e8
+            thkMin = 0.0004
+        elif strMatType == "Al-Li":
+            rhoMat = 2700
+            sigMat = 7e8
+            thkMin = 0.004
+    
+        
+        # Propellant Density switch case
+        if strPropType=="Oxygen":
+            rhoProp =  1140    # Density of propellant (kg/m3)
+        elif strPropType=="Hydrogen":
+            rhoProp = 70
+        elif strPropType == "Methane":
+            rhoProp = 420
+        elif strPropType == "MMH":
+            rhoProp = 866
+        elif strPropType == "NTO":
+            rhoProp = 1450
+        elif strPropType == "RP-1":
+            rhoProp = 820
+        
+        
+        # Calculate propellant volume and volume per tank (include ullage)
+        # volPropOx = MissionSummary.mPropImpulseOx/TankSet.Oxygen
+        # volPropFuel = MissionSummary.mPropImpulseFuel/TankSet.strPropType
+        volPropTotal = (MissionSummary.mPropImpulseOx/TankSet.rhoProp) + (MissionSummary.mPropImpulseFuel/TankSet.rhoProp)
```

Comment:
This should be mPropImpulse/TankSet.rhoProp. The way the code is set up, it's not broken down into oxidizer/fuel separately